templates/pages/paper.jsx

```diff
@@ -20,11 +20,31 @@ var Issue = React.createClass({
       this.replaceWith('home');
     }
   },
+  validateEmail: function(emailId) {
+    var emailRe = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Thanks @jinankjain! I'm not sure if a regex is the best way to handle this -- have you looked at any packages? 

This seems interesting:
https://www.npmjs.com/package/validator